qt/python/mantidqt/mantidqt/widgets/regionselector/test/test_regionselector_presenter.py

```diff
@@ -87,6 +87,27 @@ def test_add_second_rectangular_region_deactivates_first_selector(self):
         self.assertFalse(region_selector._selectors[0].active)
         self.assertTrue(region_selector._selectors[1].active)
 
+    def test_clear_workspace_will_clear_all_the_selectors_and_model_workspace(self):
+        region_selector = RegionSelector(view=Mock())
+        mock_ws = Mock()
+
+        region_selector.update_workspace(mock_ws)
+        region_selector.add_rectangular_region("test", "black")
+        region_selector.add_rectangular_region("test", "black")
+
+        region_selector.clear_workspace()
+
+        self.assertEqual(0, len(region_selector._selectors))
+        self.assertEqual(None, region_selector.model.ws)
+
```

Comment:
Does this mean that in addition to the plot clearing when two overlapping signal areas are drawn that the signal areas should also disappear?

Reply:
Not quite. This is for when there is an error when summing banks, then the 2nd plot (with the selectors on it) should be cleared. I don't know of a way to test this unfortunately without making a code change to make the SumBanks algorithm break